src/ProductConstructionService/ProductConstructionService.SubscriptionTriggerer/appsettings.Development.json

```diff
@@ -0,0 +1,4 @@
+{
+    "BuildAssetRegistrySqlConnectionString": "Server=tcp:maestro-int-server.database.windows.net,1433;Initial Catalog=BuildAssetRegistry;Encrypt=True;TrustServerCertificate=False;Connection Timeout=30;Authentication=\"Active Directory Default\";",
+    "QueueConnectionString": "https://productconstructionint.queue.core.windows.net/pcs-jobs"
```

Comment:
This should be the local Azurite URI

Reply:
yup, still need to set it, configuring some stuff in Azure now